packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/collapsible/collapsible.css

```diff
@@ -32,7 +32,7 @@
 	}
 
 	& > .ck-collapsible__children {
-		padding: 0 var(--ck-spacing-large) var(--ck-spacing-large);
+		padding: var(--ck-spacing-medium) var(--ck-spacing-large) var(--ck-spacing-large);
```

Comment:
Why top padding is changed from `large` into `medium`? When button is focused it looks odd when label in on the box-shadow outline:  
 
![Screenshot 2023-12-05 at 15 44 30](https://github.com/ckeditor/ckeditor5/assets/7074833/c3bd706d-618d-4d8e-97a1-6020fbb0468d)

But as i understand decision was made - i just wanted to show my doubts.

Reply:
It's increased from `0` to `medium` 😀

Reply:
Missed the `0` , sorry! but still, large looks a bit better IMO ;)

<img width="283" alt="Screenshot 2023-12-05 at 16 17 17" src="https://github.com/ckeditor/ckeditor5/assets/7074833/de63d4b8-b8a8-4074-994e-5a9fcbd27905">

Reply:
When it's larger it looks strange when the button outline is not visible.